mine.sh

```diff
@@ -7,7 +7,7 @@ GRIND="bitcoin-util grind"
 NBITS=${NBITS:-"1e0377ae"} #minimum difficulty in signet
 
 echo "Waiting until Chain tip age is < $CHAIN_TIP_AGE seconds before mining start..."
-wait_chain_sync.sh $CHAIN_TIP_AGE
+# wait_chain_sync.sh $CHAIN_TIP_AGE
```

Comment:
Why don't we need this now? It still says "waiting until..." in the comment just above. If it is not needed you can remove this comment.